app/script/view_model/content/InputBarViewModel.js

```diff
@@ -305,11 +327,14 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
       this.cancelMessageEditing();
       messageEntity.isEditing(true);
       this.editMessageEntity(messageEntity);
-      this.currentMentions(messageEntity.get_first_asset().mentions());
+
+      // clean mentions before setting text in order to prevent mentions offset recomputing
+      this.currentMentions.removeAll();
```

Comment:
This is an actual bug that was introduced since #4702 (the mentions where shifted because we set the mentions before setting the input)

Reply:
Is this related to reopening WEBAPP-5489?